GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -305,6 +303,7 @@ public static string RunCmd(string cmd, string arguments)
             }
         }
 
+        [Obsolete]
```

Comment:
❕ Need to add message

Reply:
All obsolete code would be removed before this PR is merged. It's just a marker for now. All these commits would be squashed and sanitised. For now I just want to know whether there's adequate interest in such a PR.